knowledge_base/Heroku.md

```diff
@@ -1,6 +1,66 @@
 # Heroku
 
-## Production Database
+Engineers should be assigned Heroku admin permissions during the onboarding process. If this is not the case, reach out to an engineering lead.
+
+## Contents
+
+- [Apps](#apps)
+  + [Common-app](#common-app)
+  + [Common-bot](#common-bot)
+- [Procfiles](#procfiles)
+- [Buildpacks](#buildpacks)
+- [Release phases](#release-phases)
+- [Downloading Production Databases](#downloading-production-databases)
+- [Change Log](#change-log)
+
+## Apps
+
+As of 231107, eight apps are hosted on Heroku:
+
+1. `common-app`
+2. `commonbot`
+3. `commonwealth-frack-eu`
+4. `cosmos-devnet`
+5. `cosmos-devnet-beta`
+6. `cw-discourse-import`
+7. `evmos-devnet`
+8. `snapshot-listener`
+
+### Common-app
+
+The `common-app` comprises several sub-apps or servers:
```

Comment:
The proper terminology here is that technically `common-app` is a pipeline and then all the apps within it are different stages of the pipeline.